hypothesis-python/src/hypothesis/internal/coverage.py

```diff
@@ -44,7 +44,7 @@ def pretty_file_name(f):
         pass
 
     parts = f.split(os.path.sep)
-    if "hypothesis" in parts:
+    if "hypothesis" in parts:  # pragma: no branch
         parts = parts[-parts[::-1].index("hypothesis") :]
```

Comment:
From what I can tell, the else-branch is only reached if we call one of the branch-tracking functions from a source file that doesn't have an exact match for `hypothesis` somewhere in its path.

Currently we only ever call these functions from files inside `hypothesis-python/src/hypothesis/`, and I don't see any value in specifically trying to cover the else.